nuget.config

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since this is all released now, should we delete _all_ the nuget.config files from this branch?